### DISCUSSION TITLE
[firebase_crashlytics] Errors recorded while testing

### DISCUSSION THREAD

**Discussion by Zazo032** [firebase/flutterfire#5591] [Feature request]
On our app, some functions have a specific Crashlytics `recordError` call. While using/testing any of those functions on our app's tests, there's a lot of verbose when expecting them to throw errors/exceptions. Is this expected behavior and are those reports being sent, or can I just ignore those logs?

```
C:\src\flutter\bin\flutter.bat --no-color test --machine test
Shuffling test order with --test-randomize-ordering-seed=0
Error caught by Crashlytics plugin <recordError>:
The following exception was thrown testException:
<StackTrace>
```

We're using version `0.1.3` of the plugin.

**Comment by creativecreatorormaybenot** [comment]
Do you mean the `Error caught` line?

**Comment by Zazo032** [comment]
Yes, that's the normal output when debugging or on release, but I'm not sure if it should appear on tests.

**Comment by creativecreatorormaybenot** [comment]
@Zazo032 I remember when rewriting the code that I just left it in. It is definitely intended behavior from the programming side. However, maybe it can be discussed if this is sensible. It is not a bug - the crash reports are not sent to Crashlytics (unless you enable `enableInDevMode`).

**Comment by Zazo032** [comment]
Thanks, that was my doubt, if I was seeing those logs because they were being sent. Maybe verbose on tests or dev mode could be disabled via a parameter? Like `Crashlytics.enableVerbose` or something similar.
